torch/csrc/jit/codegen/cuda/arith.cpp

```diff
@@ -199,6 +201,14 @@ Val* getMinimumValue(DataType v) {
     case (DataType::Float):
       return IrBuilder::create<Double>(-std::numeric_limits<float>::infinity());
       break;
+    case (DataType::Half):
```

Comment:
These are actually not necessary if we do not plan to support half math.
Feel free to yell at me if we want to take them off.